contrib/cirrus/CIModes.md

```diff
@@ -42,72 +42,33 @@ of this document, it's not possible to override the behavior of `$CIRRUS_PR`.
 
 ## Cirrus Task contexts and runtime modes
 
-### Intended general PR Tasks (*italic*: matrix)
-+ *build*
-+ validate
-+ bindings
-+ swagger
-+ *alt_build*
-+ osx_alt_build
-+ freebsd_alt_build
-+ docker-py_test
-+ *unit_test*
-+ apiv2_test
-+ *compose_test*
-+ *local_integration_test*
-+ *remote_integration_test*
-+ *container_integration_test*
-+ *rootless_integration_test*
-+ *local_system_test*
-+ *remote_system_test*
-+ *rootless_remote_system_test*
-+ *buildah_bud_test*
-+ *rootless_system_test*
-+ rootless_gitlab_test
-+ *upgrade_test*
-+ meta
-+ success
-+ artifacts
-
-### Intended for PR w/ "release" or "bump" in title:
-+ (All the general PR tasks above)
-+ release_test
-
-### Intended `[CI:DOCS]` PR Tasks:
-+ *build*
-+ validate
-+ swagger
-+ meta
-+ success
-
-### Intended `[CI:BUILD]` PR Tasks:
-+ *build*
-+ validate
-+ *alt_build*
-+ osx_alt_build
-+ freebsd_alt_build
-+ meta
-+ success
-+ artifacts
-
-### Intended `[CI:MACHINE]` PR Tasks:
-
-If and only if the PR is in **draft-mode**, run only the following
-tasks.  The draft-mode check is necessary to remove the risk of
-merging a change that affects the untested aspects of podman.
-
-+ *build*
-+ validate
-+ *alt_build*
-+ win_installer
-+ osx_alt_build
-+ podman_machine_task
-+ podman_machine_aarch64_task
-+ podman_machine_windows_task
-+ podman_machine_mac_task
-+ meta
-+ success
-+ artifacts
+By default cirrus will trigger task depending on the source changes.
+As such a user does not have to use any special titles under normal circumstances.
```

Comment:
This sentence is temporarily helpful right now, today and this week/month, as a reminder that CI:DOCS and friends are gone. A new contributor might not have any idea what "special titles" means, and even one of us, in six months, might take a moment to remember "oh yeah we had to manually do all that nonsense". I vote for removing this sentence.

Reply:
removed